packages/php-table-backend-utils/phpstan-baseline.neon

```diff
@@ -36,11 +36,26 @@ parameters:
 			path: src/Connection/ConnectionRetryWrapper.php
 
 		-
-			message: "#^Parameter \\#1 \\$params of static method Doctrine\\\\DBAL\\\\DriverManager\\:\\:getConnection\\(\\) expects array\\{charset\\?\\: string, dbname\\?\\: string, default_dbname\\?\\: string, driver\\?\\: 'ibm_db2'\\|'mysqli'\\|'oci8'\\|'pdo_mysql'\\|'pdo_oci'\\|'pdo_pgsql'\\|'pdo_sqlite'\\|'pdo_sqlsrv'\\|'sqlite3'\\|'sqlsrv', driverClass\\?\\: class\\-string\\<Doctrine\\\\DBAL\\\\Driver\\>, driverOptions\\?\\: array, host\\?\\: string, keepSlave\\?\\: bool, \\.\\.\\.\\}, array\\{port\\?\\: string, warehouse\\?\\: string, database\\?\\: string, schema\\?\\: string, tracing\\?\\: int, loginTimeout\\?\\: int, networkTimeout\\?\\: int, queryTimeout\\?\\: int, \\.\\.\\.\\} given\\.$#"
+			message: "#^Parameter \\#1 \\$params \\(array\\{host\\: string, user\\: string, password\\: string, skipCertCheck\\: bool\\}\\) of method Keboola\\\\TableBackendUtils\\\\Connection\\\\Exasol\\\\ExasolDriver\\:\\:connect\\(\\) should be compatible with parameter \\$params \\(array\\{charset\\?\\: string, dbname\\?\\: string, defaultTableOptions\\?\\: array\\<string, mixed\\>, default_dbname\\?\\: string, driver\\?\\: 'ibm_db2'\\|'mysqli'\\|'oci8'\\|'pdo_mysql'\\|'pdo_oci'\\|'pdo_pgsql'\\|'pdo_sqlite'\\|'pdo_sqlsrv'\\|'pgsql'\\|'sqlite3'\\|'sqlsrv', driverClass\\?\\: class\\-string\\<Doctrine\\\\DBAL\\\\Driver\\>, driverOptions\\?\\: array, host\\?\\: string, \\.\\.\\.\\}\\) of method Doctrine\\\\DBAL\\\\Driver\\:\\:connect\\(\\)$#"
```

Comment:
Este to zacalo padat na phpstane tak som updatol baseline https://keboola.slack.com/archives/CFVRE56UA/p1675932077551409 a davam este na review